test/e2e/metamask-ui.spec.js

```diff
@@ -1053,19 +1065,19 @@ describe('MetaMask', function () {
       await driver.clickElement('.advanced-gas-options-btn');
       await driver.delay(regularDelayMs);
 
-      gasModal = await driver.findElement('span .modal');
+      // Wait for gas modal to be visible
+      await driver.waitForSelector('span .modal');
       await driver.delay(regularDelayMs);
     });
 
     it('opens customize gas modal', async function () {
```

Comment:
Nit: Pre-existing issue, but... this description is inaccurate :thinking:

Reply:
i didn't know what to do here so I moved the steps from the prior block that actually opened the modal into this step and reworded it slightly.